pkg/cmd/grafana-cli/services/api_client.go

```diff
@@ -67,11 +67,11 @@ func (client *GrafanaComClient) DownloadFile(pluginName string, tmpFile *os.File
 				logger.Info("Failed downloading. Will retry once.")
 				err = tmpFile.Truncate(0)
 				if err != nil {
-					panic(err)
+					return
```

Comment:
Would you mind just logging the error here? Thanks.

Reply:
Ok, but as `DownloadFile` returns `err` (named return value), the error will be printed anyway by the caller (`InstallPlugin`).

Reply:
Ah you're right... I didn't notice it's setting a named return value. Then it's not really necessary.